themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Projects in Self-Managed State Backends"
+date: 2023-04-03
+meta_desc: Pulumi's Self-Managed backend now supports stack names scoped by project name.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+    - abhinav-gupta
+tags:
+    - features
+    - pulumi-releases
+---
+
+At Pulumi, we aim to offer our users the best-in-class IaC experience.
+We recently shipped a series of impactful enhancements to our [Pulumi Service](/product/pulumi-service/) backend
```

Comment:
any other examples we could link to here for recent enhancements?

Reply:
I'm aware of the next big incoming thing for the service (which I didn't want to spoil here), but there weren't too many service-specific announcements recently that I could point to here. 
I figured if I didn't have a list of 3-5 more, I shouldn't say "enhancements like A, B, C, ..., and more on the way", and instead went with suggesting that there are many enhancements—side note, check out deployments, our most recent cool feature.
That said, I haven't worked on the service at all, so I could be wrong about what we want to highlight here?